.readthedocs.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
```

Comment:
what version does this set?

Reply:
This is RTD build/backend version https://docs.readthedocs.io/en/stable/config-file/v2.html#version, not something PyMC related